x/margin/types/types.go

```diff
@@ -1,3 +1,6 @@
+//go:build FEATURE_TOGGLE_MARGIN_CLI_ALPHA
+// +build FEATURE_TOGGLE_MARGIN_CLI_ALPHA
+
```

Comment:
Are these build tags needed in all files or just one's that are imported by the build path? If the dependency path to here is disabled higher up surely these tags aren't needed?